src/plugins/bar-graph/bar-graph-tile.tsx

```diff
@@ -1,48 +1,80 @@
-import React from "react";
+import React, { useRef } from "react";
 import classNames from "classnames";
 import { observer } from "mobx-react";
 import { useResizeDetector } from "react-resize-detector";
 
+import { ChartArea } from "./chart-area";
+import { LegendArea } from "./legend-area";
 import { BasicEditableTileTitle } from "../../components/tiles/basic-editable-tile-title";
 import { ITileProps } from "../../components/tiles/tile-component";
-import { ChartArea } from "./chart-area";
 import { BarGraphModelContext } from "./bar-graph-content-context";
 import { isBarGraphModel } from "./bar-graph-content";
 import { TileToolbar } from "../../components/toolbar/tile-toolbar";
-import { LegendArea } from "./legend-area";
 
 import "./bar-graph.scss";
 
 import "./bar-graph-toolbar";
 
 const legendWidth = 190;
-const legendHeight = 190; // FIXME
 
 export const BarGraphComponent: React.FC<ITileProps> = observer((props: ITileProps) => {
-
-  const { model, readOnly } = props;
+  const { model, readOnly, onRequestRowHeight } = props;
   const content = isBarGraphModel(model.content) ? model.content : null;
 
-  const {height: containerHeight, width: containerWidth, ref} = useResizeDetector();
+  const requestedHeight = useRef<number|undefined>(undefined);
+
+  const onResize = (width: number|undefined, height: number|undefined) => {
```

Comment:
This has the potential to cause an infinite resize loop. When the `onRequestRowHeight` is handled the tile will change size which will trigger this onResize function. I'd guess the "debounce" refreshMode will prevent it from flickering too fast if this happens, but it still seems theoretically possible. However I can't come up with a case where this happens, so it is probably fine.